index.html

```diff
@@ -0,0 +1,163 @@
+            am highly adaptable and can work well under pressure whilst still
+            being organised and maintaining a high level of professionalism.
+          </p>
+        </div>
+      </section>
+
+      <!-- the second section within the main tag contains the enlarged main project card and smaller pending project cards   -->
+
+      <section class="entire-project">
+        <!-- Main project section that contains an enlarged project card img that is nested in a anchor tag  -->
+
+        <h2 id="projects" class="project-header">Projects</h2>
+
+        <div class="main-project">
+          <div class="main-img-container">
+            <a
+              href="https://dmo17.github.io/semantic-html-refactor/"
+              target="blank_"
+            >
+              <img
+                src="./assets/images/web-page-preview.png"
+                alt="preview-image-project"
+              />
+            </a>
+          </div>
+
+          <div class="project-title">
+            <h3>Semantic HTML & CSS Refactor</h3>
+          </div>
+        </div>
+
+        <!-- Placeholder img cards that is nested in a div tag which is all nested in a div tag -->
+
+        <div class="future-projects-container">
+          <div class="future-projects">
+            <img src=".//assets/images/project-icon.PNG" alt="project-icon" />
+
+            <h3 class="pending-project-title">Pending-Project Title</h3>
+          </div>
+
+          <div class="future-projects">
+            <img src=".//assets/images/project-icon.PNG" alt="project-icon" />
+
+            <h3 class="pending-project-title">Coming Soon</h3>
+          </div>
+
+          <div class="future-projects">
+            <img src=".//assets/images/project-icon.PNG" alt="project-icon" />
+
+            <h3 class="pending-project-title">Coming Soon</h3>
+          </div>
+
+          <div class="future-projects">
+            <img src=".//assets/images/project-icon.PNG" alt="project-icon" />
+
+            <h3 class="pending-project-title">Coming Soon</h3>
+          </div>
+
+          <div class="future-projects">
+            <img src=".//assets/images/project-icon.PNG" alt="project-icon" />
+
+            <h3 class="pending-project-title">Coming Soon</h3>
+          </div>
+
+          <div class="future-projects">
+            <img src=".//assets/images/project-icon.PNG" alt="project-icon" />
+
+            <h3 class="pending-project-title">Coming Soon</h3>
+          </div>
+        </div>
+      </section>
+
+      <!-- this section consists of a div tags with anchor tags linking them to websites. this is all nested in a section tag-->
+
+      <section class="contact-me-container">
+        <h2 id="contact" class="contact-me-title">Contact Me</h2>
+        <div class="contact-link-container">
+          <div class="details">
+            <h3 class="c-title">Email:</h3>
+            <a href="#">dmoabdi98@gmail.com</a>
```

Comment:
Consider looking into using a `mailto:` link to make it so when it's clicked, it opens a new email window. Have a google on how to do this.